app/src/main/java/ch/epfl/sdp/blindly/database/UserRepositoryImpl.kt

```diff
@@ -53,7 +54,7 @@ class UserRepositoryImpl constructor(
             Log.d(TAG, "Found user with uid: $uid in cache")
             return cached
         }
-        /*
```

Comment:
Is having a local db and a user cache really useful? If we are too short in time to resolve this, it is ok for me too keep it as is.

Reply:
forgot to uncomment this

Reply:
hum this is not commented on my side seem like this is not the last version that you are reviewing

Reply:
yes it is necessary because the user is persistently stored and the cache store every user that is retreived from firestore